test/fixtures/order_items.yml

```diff
@@ -2,6 +2,11 @@
 
 one:
   quantity: 1
+  product: manny
+  order: first
 
 two:
-  quantity: 1
+  quantity: 0
```

Comment:
doesn't this violate the validation for order_items? if we want to test the case that validations work, maybe this is something we would adjust within the test itself (i.e. have a product and set the quantity to 0 within the test case itself --> order_item = orders(:one) and then order_item.quantity = 0